storybook/src/components/Footer/Footer.stories.tsx

```diff
@@ -74,7 +71,7 @@ export const Default: Story = {
             </LinkList>
           </Grid.Cell>
           <Grid.Cell span={4} start={{ narrow: 1, medium: 1, wide: 9 }}>
-            <section className="ams-mb--md">
+            <div className="ams-mb--md">
```

Comment:
> It also changes a section to a div, [a section should not be used for styling purposes only](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#usage_notes).

But we don’t apply styling based on the tag name. We have content with a heading here, why not mark that up as a section?

Reply:
The elements function seems to be to vertically separate 2 sections in a `Grid.Cell`. That's a purely visual thing.

[Using too many landmarks can create 'noise' for screen readers](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/region_role#accessibility_concerns).